radixdlt-core/docker/scripts/generate-yml.sh

```diff
@@ -6,17 +6,41 @@ set -e
 # Where we are run from
 scriptdir=$(dirname "$0")
 
-# Number of validators
-validators=${1:-1}
+while getopts fn:p: flag
+do
+  case "${flag}" in
+    f) force_delete=true;;
+    p) port_param=${OPTARG};;
+    n) validators=${OPTARG};;
+    *) echo "Invalid parameters … exiting"
```

Comment:
I think it would be a good idea to add a -h parameter to explain the syntax.
Then instead of printing "Invalid parameters", print the same message as -h

Reply:
Makes a lot of sense. Yup, I'll do that.